CONTRIBUTING.md

```diff
@@ -40,6 +40,138 @@ GitHub provides additional document on [forking a repository](https://help.githu
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
 
+### Running Tests in a local Environment (Docker Compose)
+
+#### Unit Tests
+
+To verify unit tests pass run:
+
+```bash
+mvn clean test
+```
+
+#### Formatting
+
+The CI pipeline will verify that source code files are correctly formatted (using `com.spotify.fmt:fmt-maven-plugin:check`) and license headers are present (using )
+
+To verify correct formatting of your code run:
+
+```bash
+mvn com.spotify.fmt:fmt-maven-plugin:check
+mvn com.mycila:license-maven-plugin:check
+```
+
+> **Note**  
+> When you run `mvn package` the formatter will auto-format files.
+
+#### Packaging
+
+To create the (shaded) JAR(s) run:
+
+```bash
+mvn clean package -Drevision=$(git describe --tags --always)
+```
+
+#### Integration Tests with Docker Compose
+
+To execute the integration tests (requires [Docker](https://docker.com/)) run:
```

Comment:
Is there an alternative we can offer for organizations greater than 250 who do not have a license https://www.docker.com/pricing/faq/ 

> Docker Desktop requires a paid, per-user subscription for organizations with more than 250 employees or more than $10 million in annual revenue per our [terms of service](https://www.docker.com/legal/docker-terms-service/).

Reply:
Good catch! Will ammend!